apps/web/src/pages/Communities/UpdateCommunityPage/UpdateCommunityPage.tsx

```diff
@@ -0,0 +1,371 @@
+            </Link>
+          </div>
+        </CardBasic>
+      </form>
+    </FormProvider>
+  );
+};
+
+interface RouteParams extends Record<string, string> {
+  communityId: Scalars["ID"]["output"];
+}
+
+const UpdateCommunityPage_Query = graphql(/* GraphQL */ `
+  query UpdateCommunityPage($id: UUID!) {
+    community(id: $id) {
+      name {
+        en
+        fr
+      }
+
+      ...UpdateCommunityPage_Community
+    }
+  }
+`);
+
+const UpdateCommunity_Mutation = graphql(/* GraphQL */ `
+  mutation UpdateCommunity($id: UUID!, $community: UpdateCommunityInput!) {
+    updateCommunity(id: $id, community: $community) {
+      id
+    }
+  }
+`);
+
+export const UpdateCommunity = () => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const { communityId } = useRequiredParams<RouteParams>("communityId");
+  const [{ data, fetching, error }] = useQuery({
+    query: UpdateCommunityPage_Query,
+    variables: { id: communityId || "" },
+  });
+
+  const [{ fetching: isSubmitting }, executeMutation] = useMutation(
+    UpdateCommunity_Mutation,
+  );
+
+  const { navigationCrumbs: baseCrumbs } = useOutletContext<ContextType>();
+
+  const handleSave = async (input: UpdateCommunityInput) => {
+    return executeMutation({ id: communityId, community: input }).then(
+      (result) => {
+        if (result.data?.updateCommunity) {
+          return Promise.resolve();
+        }
+        return Promise.reject(new Error(result.error?.toString()));
+      },
+    );
+  };
+
+  const crumbs = [
+    ...(baseCrumbs ?? []),
+    {
+      label: intl.formatMessage({
+        defaultMessage: "Edit<hidden> skill</hidden>",
+        id: "M2LfhH",
+        description: "Breadcrumb title for the edit skill page link.",
+      }),
+      url: paths.communityUpdate(communityId),
+    },
+  ];
+
+  const pageTitle = intl.formatMessage({
+    defaultMessage: "Edit a community",
+    id: "OY5/aW",
+    description: "Page title for the edit community page.",
+  });
+
+  return (
+    <>
+      <SEO title={pageTitle} />
+      <Hero title={pageTitle} crumbs={crumbs} overlap centered>
+        <div data-h2-margin-bottom="base(x3)">
+          <Pending fetching={fetching} error={error}>
+            {data?.community ? (
+              <CommunityForm
+                query={data.community}
+                onUpdate={handleSave}
+                isSubmitting={isSubmitting}
+              />
+            ) : (
+              <NotFound
+                headingMessage={intl.formatMessage(commonMessages.notFound)}
+              >
+                <p>
+                  {intl.formatMessage(
+                    {
+                      defaultMessage: "Skill {skillId} not found.",
+                      id: "953EAy",
+                      description: "Message displayed for skill not found.",
+                    },
+                    { skillId: communityId },
+                  )}
+                </p>
```

Comment:
Need to update message to refer to community instead of skill

Reply:
Oops!  :sweat_smile: Good catch.
[fix error message](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12005/commits/9dfb23d4699a480bd96cb93825b51bce8abef104)